pypeit/core/gui/identify.py

```diff
@@ -801,6 +802,224 @@ def store_solution(self, final_fit, binspec, rmstol=0.15,
         # For the cases that need the wvarxiv name, return it
         return wvarxiv_name
 
+    def store_solution_multi(self, final_fit, binspec, rmstol=0.15,
+                       force_save=False, wvcalib=None, multi = False, 
+                       fits_dicts = None, specdata = None, slits = None, 
+                       lines_pix_arr = None, lines_wav_arr = None, lines_fit_ord = None, 
+                       custom_wav = None, custom_wav_ind = None):
+        """Check if the user wants to store this solution in the reid arxiv, when doing the wavelength solution
+        for multiple traces
+
+        Parameters
+        ----------
+
+        final_fit : dict
+            Dict of wavelength calibration solutions (see self.get_results())
+        binspec : int
+            Spectral binning
+        rmstol : float
+            RMS tolerance allowed for the wavelength solution to be stored in the archive
+
+        Optional Parameters
+        -------------------
+
+        force_save : bool
+            Force save
+        multi : bool
+            Is the template multiple traces?
+        fits_dict : list
+            List of dictionaries containing the _fitdict of previous calls, if multi-trace data
+        specdata : array
+            Numpy array containing the flux information from all the traces
+        wvcalib : :class:`pypeit.wavecalib.WaveCalib`
+            Wavelength solution
+        lines_pix_arr : array
+            Numpy array containing the pixel locations of all ID'd lines
+        lines_wav_arr : array
+            Numpy array containing wavelengths of all the ID'd lines
+
+        Returns
+        -------
+
+        wvarxiv_name : :obj:`str`
+            The name of the wvarxiv file if saved. "None" if not saved
+        """
+        # For return
+        wvarxiv_name = None
+        
+        # Solution
+        if 'rms' not in final_fit.keys():
+            msgs.warn("No wavelength solution available")
+            return
+        elif final_fit['rms'] < rmstol or multi:
+            #ans = input(f"rms = {final_fit['rms']}, which is greater than rmstol = {rmstol}! Save anyway? (y/[n]): ")
+            ans = ''
+            if not force_save:
+                while ans != 'y' and ans != 'n':
+                    ans = input("Would you like to write this wavelength solution to disk? (y/n): ")
+            else:
+                ans = 'y'
+            if ans == 'y':
+                # Arxiv solution
+                if multi:
+                    # prompt the user to give the orders that were used here
+                    if '"echelle": true' in wvcalib.strpar:
+                        while True:
+                            try:
+                                print('    ')
+                                order_str = input("Which orders were we fitting? e.g. (32:39):  ")    
```

Comment:
I have used "input" because it is what was being used in pypeit_identify before I added this part of the code. Why is adding a msgs.input() call here better than just the input()?

Reply:
It's not essential, it just has a pretty message. I think this can be ignored, as there are lots of instances of the missing `msgs.input()` in this script that predates your updates 👍